javascript/builtins/Error.json

```diff
@@ -228,6 +228,56 @@
             }
           }
         },
+        "captureStackTrace": {
+          "__compat": {
+            "tags": [
+              "web-features:javascript"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": "17"
+              },
+              "chrome_android": "mirror",
+              "deno": {
+                "version_added": false
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "136",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "javascript.options.experimental.error_capture_stack_trace",
+                    "value_to_set": "true"
+                  }
+                ],
+                "impl_url": "https://bugzil.la/1886820"
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "nodejs": {
+                "version_added": "16.17.0"
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "17.3"
+              },
+              "safari_ios": "mirror",
+              "samsunginternet_android": "mirror",
+              "webview_android": "mirror",
+              "webview_ios": "mirror"
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": false,
```

Comment:
This is expected to be in a TC39 proposal in Feb (this month). At that point we can add proposal as a spec URL presumably.